daprdocs/content/en/developing-applications/integrations/Azure/azure-authentication/howto-mi.md

```diff
@@ -10,9 +10,55 @@ description: "Learn how to use Managed Identities"
 
 Using Managed Identities (MI), authentication happens automatically by virtue of your application running on top of an Azure service that has an assigned identity. 
 
-For example, let's say you enable a managed service identity for an Azure VM, Azure Container App, or an Azure Kubernetes Service cluster. When you do, an Microsoft Entra ID application is created for you and automatically assigned to the service. Your Dapr services can then leverage that identity to authenticate with Microsoft Entra ID, transparently and without you having to specify any credentials.
+Let's say you enable a managed service identity using an Azure KeyVault secrets component for an Azure service. When you do, an Microsoft Entra ID application is created for you and automatically assigned to the service. Your Dapr services can then leverage that identity to authenticate with Microsoft Entra ID, transparently and without you having to specify any credentials.
 
-To get started with managed identities, you need to assign an identity to a new or existing Azure resource. The instructions depend on the service use. Check the following official documentation for the most appropriate instructions:
+> Dapr supports both system-assigned and user-assigned identities.
+
+In your component YAML, the [`azureClientId` property]({{< ref "authenticating-azure.md#authenticating-with-managed-identities-mi" >}}) only needs to be set if using user-assigned identity. Otherwise, you can omit this property for system-managed identity to be used by default.
+
+## System-assigned
+
+If you set up system-assigned MI using an Azure KeyVault component, the YAML would look like the following:
+
+```yml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: azurekeyvault
+spec:
+  type: secretstores.azure.keyvault
+  version: v1
+  metadata:
+  - name: vaultName
+    value: mykeyvault
+```
+
+In this example, the system-assigned MI looks up the service identity and communicates with the `mykeyvault` vault.
+
+## User-assigned
+
+If you set up user-assigned MI using an Azure KeyVault component, the YAML would look like the following:
```

Comment:
I would keep managed ONLY for managed identities and otherwise simply say user assigned identities.
```suggestion
If you set up user-assigned identity using an Azure KeyVault component, the YAML would look like the following:
```